src/lib/ollama.ts

```diff
@@ -28,5 +9,18 @@ export async function ollamaTags() {
 	const response = await fetch(`${settings.ollamaServer}/api/tags`);
 	if (!response.ok) throw new Error('Failed to fetch Ollama tags');
 
-	return response.json() as Promise<OllamaTagResponse>;
+	const data: ListResponse | undefined = await response.json();
+	if (!data || !Array.isArray(data.models)) {
+		throw new Error('Failed to parse Ollama tags', { cause: data });
+	}
+
+	// Sort alphabetically
+	data.models = data.models.sort((a, b) => {
+		const nameA = a.name;
+		const nameB = b.name;
```

Comment:
There as a mistake in `type OllamaModel` where `model: string` didn't exist, so I updated the compare function to `a.name`/`b.name`.

We are now using the type definitions directly from `ollama-js`, in this case `ListResponse`.